eventually/event/models.py

```diff
@@ -302,4 +303,4 @@ def delete_by_id(event_id):
             event.delete()
             return True
         except (Event.DoesNotExist, AttributeError):
-            pass
+            LOGGER.error('Certain event does not deleted')
```

Comment:
'does not' or 'was not' ?
probably you can show id of that event or sth like this?

Reply:
fixed typo and added entity id to the log message for more meaningly log.